.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
+max-line-length = 89
+extend-ignore = E203
```

Comment:
I vote for the max-line-length of 89 because it is what we have in the circle ci lint. I need to fix the ignores though. Note black suggestes 88, but says anything around 90 should be good. I tried 88 but it changed too many lines of code.